package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digitalcredentials/sign-and-verify-core",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
No need to update the package.json version in PRs, that gets done automatically at publish time.

Reply:
Yep, reverted this back to `2.1.0` 👍🏾